### PR TITLE
Ensure transaction activity log supports EIP-1559

### PR DESCRIPTION
**Pull request by danjm**
Updates transaction activity log so that it shows correct values for EIP-1559 transactions

Before:

Screen Shot 2021-08-06 at 9 12 27 AM

After:

Screenshot from 2021-08-06 17-58-33
Screenshot from 2021-08-06 17-58-14
Screenshot from 2021-08-06 17-57-46

### CODE REVIEW

**Comment by darkwing** [comment]
I'm seeing the values as expected!  Well done!

**Review by darkwing — DISMISSED** [review]

File: ui/components/app/transaction-activity-log/transaction-activity-log.util.js [hunk 1]
```diff
@@ -53,11 +55,19 @@ export function getActivities(transaction, isFirstTransaction = false) {
...
+    txParams: {
+      gas: paramsGasLimit,
+      maxPriorityFeePerGas: paramsMaxPriorityFeePerGas,
+    },
+    txReceipt: { status } = {},
     type,
+    estimatedBaseFee: paramsEstimatedBaseFee,
   } = transaction;
 
+  const paramsEip1559Price =
```
Comment (adonesky1) [inline ui/components/app/transaction-activity-log/transaction-activity-log.util.js:67]: nit: could be paramsEip1559GasPrice

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Review by darkwing — APPROVED** [review]

**Review by noncash15 — CHANGES_REQUESTED** [review]
Duplicate of #

**Pull request merged**